### DISCUSSION TITLE
New version for EE WiFi?

### DISCUSSION THREAD

**Discussion by redstamp101** [aidanmacgregor/EE_WiFi-BT_WiFi-Autologin-Windows.EXE#2] [Q&A]
Are there any plans for creating any apps that could be used to auto-login to the new EE WiFi captive portal?
Thanks.

**Comment by aidanmacgregor** [comment]
What platform and with an EE broadband account?

**Reply by redstamp101** [reply]
For me this would be Android and BT broadband account.  I have been trying using Android app called "WiFi Web Login" and changing default web page and other settings to avoid the first sequence of clicks (ie <Login>, then <Choose account type>), which seem to be the issue, but without success and was therefore looking for alternatives.  BT used to have an app that was usable when it was BT WiFi, but plenty of forum posts about this now being a PITA now that there's no auto-connect capability.  Hence searching for other solutions when I can across your project.  Thanks.

**Reply by redstamp101** [reply]
Also the forums seem to concur with my issue of not being able to use on a laptop, as Wi-Fi settings just say can't connect or similar.
